### ISSUE TITLE
Installation conflict with perl

### ISSUE THREAD

**Issue by castillohair** [taborlab/FlowCal#204]
On windows with strawberry perl installed, `pip` can refer to a `.bat` script or python's `pip.exe`, which is what we want. Therefore, installation of FlowCal fails.

**Comment by castillohair** [comment]
Apparently, the following can fix the issue:

```
pip.exe install -r requirements.txt --no-cache-dir
```

However, there could also be another `pip` in the path. Also, this wouldn't solve the equivalent issue in OSX. I think the following might be better:

```
python -m pip install -r requirements.txt --no-cache-dir
```

But I haven't tested it yet.

**Comment by castillohair** [comment]
Implemented in f8b05c8eeba9fd7956e65541a0f8d77cedb47f6f. Works on Windows. Still need to test on OSX.

**Comment by castillohair** [comment]
Works in OSX.
